### ISSUE TITLE
Expand /**|*/

### ISSUE THREAD

**Issue by rchl** [20Tauri/DoxyDoxygen#166]
In a JS file, this expands to a JSDoc comment on pressing enter (`|` denotes the caret position):

```js
/**|
export function test(arg) {
}
```

but this doesn't:

```js
/**|*/
export function test(arg) {
}
```

To add a block comment I tend to press the default shortcut `option+command+/` (on Mac) to create a block comment and then type one more `*` which ends up with the second case. But DoxyDoxygen doesn't expand second case to a JSDoc comment.

**Comment by 20Tauri** [comment]
This is the expected behavior:
Within a comment, <kbd>enter</kbd> continues the comment

Your workflow is interesting and can be optimized, try [Super+Alt+Q](http://20tauri.free.fr/DoxyDoxygen/v2/page_customization.php#key-bindings-on-os-x) shortcut, which will do exactly what you want in one command.

I know this shortcut is probably not the best (https://github.com/20Tauri/DoxyDoxygen/issues/5), but it can be remapped. Let me know if it's the reason why you don't use it, perhaps other users may be affected

**Comment by rchl** [comment]
Right. Then let me turn it around a bit.

What I would actually expect is for pressing enter to continue the comment in this case, not create a JSDoc.

What happens now is that this:

```js
/**|*/
```

ends up as

```js
/**
 |*/
```

While I would expect:

```js
/**
 *|
 */
```

**Comment by 20Tauri** [comment]
This is a good suggestion. Much more complex to implement than it seems because the syntaxes are not very consistent and can vary with time (the result in JavaScript, Pascal and Matlab are very different for example)

A modified version should however be available in the middle of next week in the Evolution channel

**Comment by 20Tauri** [comment]
Now released on evolution channel

**Comment by rchl** [comment]
Doesn't seem quite right. It's a bit misaligned after pressing enter.

https://user-images.githubusercontent.com/153197/188713070-f4e0904e-c2d7-4e9a-8e32-4305feeda3bc.mov

**Comment by 20Tauri** [comment]
Thank you for the report
Fixed in 0.81.1

**Comment by rchl** [comment]
Seems perfect now, even with extra whitespace. Thank you.
